packages/kiota_http/lib/src/http_client_request_adapter.dart

```diff
@@ -80,7 +80,7 @@ class HttpClientRequestAdapter implements RequestAdapter {
   }
 
   void _setBaseUrl(RequestInformation requestInfo) {
-    requestInfo.pathParameters['baseUrl'] = baseUrl;
+    requestInfo.pathParameters['baseurl'] = baseUrl;
```

Comment:
Why this change?

Reply:
You used 'baseurl' everywhere else in the code, so I changed it.
For example in request_information_test.dart:  requestInfo.pathParameters['baseurl'] = 'http://localhost';